python/Google.py

```diff
@@ -41,6 +41,7 @@
     'admin'   : 'https://www.googleapis.com/auth/admin.directory.group',
     'group'   : 'https://www.googleapis.com/auth/apps.groups.settings',
     'reports' : 'https://www.googleapis.com/auth/admin.reports.audit.readonly',
+    'calendar' : 'https://www.googleapis.com/auth/calendar',
```

Comment:
This is a total nit pick, sorry: note that all the colons are lined up in this dictionary (and the previous dictionary).  You should probably stay consistent to that style; i.e., make all the other colons line up with the new/longer word `calendar`.